### PR TITLE
Added GPU Board Temperature Metric

### PR DESCRIPTION
**Pull request by vaidyan1-int**
Added GPU board temperature metric to Temperature

### CODE REVIEW

File: scripts/sysman/temperature.yml [hunk 1]
```diff
@@ -21,12 +21,18 @@ etors:
       desc: "The maximum temperature across all sensors in the GPU"
     - name: MEMORY
       desc: "The maximum temperature across all sensors in the local memory"
+    - name: GPU_BOARD
```
Comment (aravindksg) [inline scripts/sysman/temperature.yml:24]: since we have "MIN" on the other metric, this could be "GPU_BOARD_MAX" ?

**Comment by wdamon-intel** [comment]
@vaidyan1-int please create an issue to associate with this PR.

File: scripts/sysman/temperature.yml (hunk 1 above)
Comment (vaidyan1-int) [reply scripts/sysman/temperature.yml:24]: Agreed. The existing enum values didnt have this either - hence kept it to look uniform for all enum values.

Comment (jandres742) [reply scripts/sysman/temperature.yml:24]: new enums need to go at the bottom of the list.

Comment (aravindksg) [reply scripts/sysman/temperature.yml:24]: would it be good to expose both front and back temps here? i.e, GPU_BOARD_FRONT, GPU_BOARD_BACK and equivalent _MIN enums)?

**Comment by aravindksg** [comment]
adding reviewers: @saik-intel @sanilkumar0 @joshuaranjan @jitendrasharma1989

File: scripts/sysman/temperature.yml [hunk 2]
```diff
@@ -21,12 +21,18 @@ etors:
...
+      desc: "The maximum temperature across all sensors in the GPU Board"
+      version: "1.7"  
     - name: GLOBAL_MIN
       desc: "The minimum temperature across all device sensors"
     - name: GPU_MIN
       desc: "The minimum temperature across all sensors in the GPU"
     - name: MEMORY_MIN
       desc: "The minimum temperature across all sensors in the local device memory"
+    - name: GPU_BOARD_MIN
+      desc: "The minimum temperature across all sensors in the GPU Board"
```
Comment (saik-intel) [inline scripts/sysman/temperature.yml:34]: IS MIN temperature really required ? Earlier we removed this support, MAX temp is interesting telemetry to aware, is the min temp requirement ?

Comment (aravindksg) [reply scripts/sysman/temperature.yml:34]: for now the spec changes here maintains the symmetry (to have MAX and MIN values). implementation can choose to expose only MAX right?

File: scripts/sysman/temperature.yml (hunk 1 above)
Comment (vaidyan1-int) [reply scripts/sysman/temperature.yml:24]: There are 2 board temperature sensors. One of them will be MIN and the other MAX.

File: scripts/sysman/temperature.yml (hunk 2 above)
Comment (vaidyan1-int) [reply scripts/sysman/temperature.yml:34]: There are 2 board sensors - one will be MIN and one will be MAX

File: scripts/sysman/temperature.yml (hunk 1 above)
Comment (aravindksg) [reply scripts/sysman/temperature.yml:24]: Overall the PR lgtm. +1

File: scripts/sysman/temperature.yml [hunk 3]
```diff
@@ -21,12 +21,18 @@ etors:
       desc: "The maximum temperature across all sensors in the GPU"
     - name: MEMORY
       desc: "The maximum temperature across all sensors in the local memory"
+    - name: GPU_BOARD
+      desc: "The maximum temperature across all sensors in the GPU Board"
+      version: "1.7"
```
Comment (wdamon-intel) [inline scripts/sysman/temperature.yml:26]: please change to 1.8

File: scripts/sysman/temperature.yml [hunk 4]
```diff
@@ -21,12 +21,18 @@ etors:
...
+      version: "1.7"  
     - name: GLOBAL_MIN
       desc: "The minimum temperature across all device sensors"
     - name: GPU_MIN
       desc: "The minimum temperature across all sensors in the GPU"
     - name: MEMORY_MIN
       desc: "The minimum temperature across all sensors in the local device memory"
+    - name: GPU_BOARD_MIN
+      desc: "The minimum temperature across all sensors in the GPU Board"
+      version: "1.7"
```
Comment (wdamon-intel) [inline scripts/sysman/temperature.yml:35]: please change to 1.8

Comment (vaidyan1-int) [reply scripts/sysman/temperature.yml]: Done

**Review by aravindksg — APPROVED** [review]

**Review by wdamon-intel — APPROVED** [review]

File: scripts/sysman/temperature.yml (hunk 1 above)
Comment (jandres742) [inline scripts/sysman/temperature.yml:24]: new enums values need to be added at the end, to avoid breaking compatibility.

**Comment by vaidyan1-int** [comment]
Updated enum

**Review by sanilkumar0 — APPROVED** [review]

**Review by jandres742 — APPROVED** [review]

**Pull request merged**